src/main/deltachat/burnerAccounts.ts

```diff
@@ -1,4 +1,3 @@
-import fetch from 'node-fetch'
```

Comment:
nah that line belongs here, I guess you forget an `npm install` to install the typings...

Reply:
Ouups :D